OpenEdXMobile/res/layout/fragment_locked_course_unit.xml

```diff
@@ -0,0 +1,90 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/white"
+        android:gravity="center"
+        android:orientation="vertical">
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <android.support.constraint.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@drawable/payment_blue_bg_2"
+                    android:paddingLeft="@dimen/edx_margin"
+                    android:paddingTop="@dimen/edx_double_margin"
+                    android:paddingRight="@dimen/edx_margin"
+                    android:paddingBottom="@dimen/edx_double_margin">
+
+                    <org.edx.mobile.view.custom.IconImageViewXml
+                        android:id="@+id/lock"
+                        android:layout_width="@dimen/fa_xxx_large"
+                        android:layout_height="@dimen/fa_xxx_large"
+                        app:iconColor="@color/white"
+                        app:iconName="fa-lock"
```

Comment:
icon is different from the available design.

Reply:
Its acceptable by client